### PR TITLE
add information about pam service name

### PR DESCRIPTION
**Pull request by rjmateus**
Signed-off-by: Ricardo Mateus <rmateus@suse.com>

# Description

Adds some clarification about rules when specifying pam service name (filename).

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [X] Manager-4.2
- [X] Manager-4.1
- [ ] Manager-4.0

# Links

### CODE REVIEW

**Comment by rjmateus** [comment]
ping @mcalmer for awareness

File: modules/administration/pages/auth-methods-pam.adoc [hunk 1]
```diff
@@ -9,6 +9,7 @@ PAM is a suite of libraries that allows you to integrate {productname} with a ce
 
 .Procedure: Enabling PAM
 . Create a PAM service file at [path]``/etc/pam.d/susemanager``.
+  Filename must be in lower case and readable by ``tomcat`` user.
```
Comment (mcalmer) [inline modules/administration/pages/auth-methods-pam.adoc:12]: I think I know what was going wrong for this customer. We use `/sbin/unix2_chkpwd` to do the pam authentication.
That bin should have the suid bit set:
```
$> ls -la /sbin/unix2_chkpwd
-rwsr-xr-x 1 root shadow 14792 19. Mai 14:13 /sbin/unix2_chkpwd*
```
I assume with this bit set, you do not need ownership to tomcat for this file.
The customer might have applied paranoid permissions settings.

Comment (rjmateus) [reply modules/administration/pages/auth-methods-pam.adoc:12]: Hey @mcalmer 
I had asked the customer to change read permissions on the file and that worked: https://bugzilla.suse.com/show_bug.cgi?id=1188558#c7

I also tested it locally and If the user that is running the pam check doesn't have read permissions on the file will get an error saying the service doesn't exist.
I think the `tomcat` user does not need to have ownership of the file but needs to be able to read it.

**Review by keichwa — APPROVED** [review]

File: modules/administration/pages/auth-methods-pam.adoc (hunk 1 above)
Comment (keichwa) [reply modules/administration/pages/auth-methods-pam.adoc:12]: @mcalmer can you please take a look again now?

Comment (mcalmer) [reply modules/administration/pages/auth-methods-pam.adoc:12]: If it was tested like this it should be ok. On our test server this file has the following permissions:
```
-rw-r--r-- 1 root root 165  1. Jul 2020  /etc/pam.d/susemanager
```
which is in line with what we write now.

**Review by mcalmer — APPROVED** [review]

**Comment by keichwa** [comment]
Thanks for all your help!  I think now I can backport it.  Or do you want to do it yourself, Ricardo?

**Comment by rjmateus** [comment]
> Thanks for all your help! I think now I can backport it. Or do you want to do it yourself, Ricardo?

Hey  @keichwa 
I just opened the PR's for manager 4.1 and 4.2.

**Comment by keichwa** [comment]
> I just opened the PR's for manager 4.1 and 4.2.

Thank you, Ricardo!  I reviewed and merged them now.

**Pull request merged**